packages/reassure-cli/src/commands/measure.ts

```diff
@@ -22,6 +24,19 @@ export function run(options: MeasureOptions) {
   const outputFile = options.baseline ? BASELINE_FILE : RESULTS_FILE;
   rmSync(outputFile, { force: true });
 
+  const metadata = {
+    metadata: {
+      branch: options.branch,
+      commitHash: options.commitHash,
+    },
+  };
+
+  const metaDataString = JSON.stringify(metadata) + '\n';
+
+  writeFileSync(outputFile, metaDataString);
+
+  console.log('');
```

Comment:
Is there a reason for adding this console empty line here?

Reply:
Sorry I will remove this right away

Reply:
Have addressed this in latest commits.